src/plugins/ssh/index.ts

```diff
@@ -242,6 +243,8 @@ const createCommand = (
 ) => {
   const commonArgs = [
     ...(args.debug ? ["-v"] : []),
+    "-i",
+    PRIVATE_KEY_PATH, // ENG-2319 Explicitly specify which private key to use
```

Comment:
This is a mega-nit but it may look like this is a TODO if the issue number is in the comment?
```suggestion
    PRIVATE_KEY_PATH, // Explicitly specify which private key to use to avoid "Too many authentication failures" caused by trying every key
```

Reply:
Done